### PR TITLE
Add mail backend for Exchange account

### PR DESCRIPTION
**Pull request by forscher21**

### CODE REVIEW

File: i3pystatus/mail/ews.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                        credentials=credentials,
+                        autodiscover=True,
+                        access_type=exchangelib.DELEGATE)
+            yield
+        except EWS_EXCEPTIONS as e:
+            # NOTE(sileht): retry just once if the connection have been
+            # broken to ensure this is not a sporadic connection lost.
+            # Like wifi reconnect, sleep wake up
+            # Wait a bit when disconnection occurs to not hog the cpu
+            print(e)
```
Comment (facetoe) [inline i3pystatus/mail/ews.py]: You should use the logger as this output will cause i3bar to fail on invalid JSON.

File: i3pystatus/mail/ews.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import exchangelib
+import contextlib
+import time
+
+from i3pystatus.mail import Backend
+
+
+EWS_EXCEPTIONS = (Exception)
```
Comment (facetoe) [inline i3pystatus/mail/ews.py]: Is this necessary considering there is only one exception?

File: i3pystatus/mail/ews.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+
+class EWS(Backend):
```
Comment (facetoe) [inline i3pystatus/mail/ews.py]: No biggie, but a more descriptive class name would be good.

File: i3pystatus/mail/ews.py (hunk 2 above)
Comment (enkore) [reply i3pystatus/mail/ews.py]: Note that EWS_EXCEPTIONS is not a tuple here, because `(1) == 1` in Python.

**Pull request merged**